### PR TITLE
feat: add unstoppable as a wallet connector

### PR DESCRIPTION
**Pull request by 0xfuje**
### This PR combined with an Apeswap frontend PR adds Unstoppable Domains as a login option

Link to frontend PR: https://github.com/ApeSwapFinance/apeswap-frontend/pull/1134

## Modified files
- src/widgets/WalletModal/types.ts to add correct typings to WalletModal
- src/widgets/WalletModal/icons/Unstoppable.tsx to add svg logo
- src/widgets/WalletModal/config.tsx to configure wallet's title, logo and connectorId

### CODE REVIEW

**Comment by obiedobo** [comment]
Thanks for submitting this and so so sorry for the extended delay.

Upon pulling your changes locally and testing, it appears there are a few issues:
1. The Unstoppable icon is not appearing correctly in storybook for me, looks like a purple rectangle.
2. A test is failing. You might need to adjust `walletmodal.test.tsx` to ensure your changes are compatible with our tests. 

You can explore the UI locally with `yarn storybook` and test with `yarn test` as needed to ensure it works before final review!

**Comment by babadrape** [comment]
> Thanks for submitting this and so so sorry for the extended delay.
> 
> Upon pulling your changes locally and testing, it appears there are a few issues:
> 
> 1. The Unstoppable icon is not appearing correctly in storybook for me, looks like a purple rectangle.
> 2. A test is failing. You might need to adjust `walletmodal.test.tsx` to ensure your changes are compatible with our tests.
> 
> You can explore the UI locally with `yarn storybook` and test with `yarn test` as needed to ensure it works before final review!

@0xfuje Awesome work with the Unstoppable Domains integration 🤩❤️😍.  I'll be picking this up along with the frontend integration and drive to completion.

File: src/widgets/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -177,7 +178,7 @@ const Navbar: React.FC<NavProps> = ({
                   onMouseLeave={() => handleHover("")}
                 >
                   {link.href ? (
-                    <MenuLink href={link.href}>
+                    <MenuLink href={link.href} target={link.label === "Lend" ? "_blank" : "_parent"}>
```
Comment (Harambe-Nakamoto) [inline src/widgets/Navbar/Navbar.tsx:181]: This causes a refresh on Exchange which we can't have

**Review by Harambe-Nakamoto — CHANGES_REQUESTED** [review]
Using _parent causes a refresh on other nav tabs

**Comment by babadrape** [comment]
> Using _parent causes a refresh on other nav tabs

Will fix this... 

Also, I noticed working with this branch doesn't let me push the UIKit Tags, so I'll be closing this PR and opening another one and make the changes on that so I can publish as well.

**Pull request closed without merging**